doc/api/Track_Selection/getAudioTrack.md

```diff
@@ -92,14 +126,37 @@ audio tracks API in the browser, this method returns "undefined".
 // Get information about the currently-playing audio track
 const audioTrack = player.getAudioTrack();
 
+// Also include metadata on the non-playable Representations
+const audioTrack = player.getAudioTrack({
+  filterPlayableRepresentations: false,
+});
+
```

Comment:
Or maybe it's because `getAudioTrack` returns an adaptation with multiple representations that may individually be supported or not?

Reply:
You're maybe mixing up "track" (RxPlayer's internal Adaptation) and Representation?

An audio track or video track can have both supported and unsupported (or decipherable and undecipherable) Representation.
When setting this property to `false`, you also get those as objects in the `representations` property, whereas by default you don't, they've been filtered out by the API.

If there's no audio track active, you still get `null`.

If there's no **supported** audio track however, and if there was audio Adaptation as inferred from the Manifest, loading will fail anyway either on a `UNSUPPORTED_CODECS_ERROR` error (or something like that, if no codec is supported) or on a `NO_PLAYABLE_REPRESENTATION` error if nothing is decipherable.

Reply:
I find the `Adaptation` concept in the RxPlayer's code very confusing anyway as it doesn't even perfectly map with DASH's AdaptationSet (e.g. they are merged if there's an `adaptation-set-switching` rule).

IMO it would be better if `track` was used everywhere as it's a clearer concept that seems to completely apply here. But this is a big change.

Reply:
Ok I understand so it make sense to have `filterPlayableRepresentations` for getAudioTrack/getVideoTrack if they can have representations that are not supported